lib/rubocop/cop/style/global_method.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop looks for definitions of global methods.
```

Comment:
Can you document the reason why it is disabled by default?
https://github.com/rubocop/rubocop/issues/9734#issuecomment-826048992

Reply:
I'd probably add a bit more rationale about the cop in general here. I guess some people might not realize where those top-level definitions end up and why there are problematic.